src/core/tsi/ssl_transport_security.cc

```diff
@@ -2061,9 +2061,6 @@ tsi_result tsi_create_ssl_client_handshaker_factory_with_options(
 
   if (factory == nullptr) return TSI_INVALID_ARGUMENT;
   *factory = nullptr;
-  if (options->pem_root_certs == nullptr && options->root_store == nullptr) {
```

Comment:
Is there a time where we do still want this return? Perhaps if `skip_server_certificate_verification == false` or something?

Reply:
Good call, done.